test/scripts/generate_script_tests.py

```
@@ -457,22 +542,44 @@ def replace_pushdata1(match):
         data_hex = match.group(2)
         return f"[1.{data_hex}]"
     script = re.sub(pushdata1_pattern, replace_pushdata1, script, flags=re.IGNORECASE)
-    
-    # PUSHDATA2: 2 bytes size specification  
+
+    # PUSHDATA2: 2 bytes size specification
     pushdata2_pattern = r'PUSHDATA2\s+0x([0-9a-fA-F]{1,4})\s+0x([0-9a-fA-F]+)'
     def replace_pushdata2(match):
         size_hex = match.group(1)
```

Ruff (0.15.4)

[error] Local variable `size_hex` is assigned to but never used

Remove assignment to unused variable `size_hex`

(F841)